ex2_hal/iris/hardware_interface/include/iris.h

```diff
@@ -28,9 +28,11 @@
 #define HOUSEKEEPING_SIZE 23 // In bytes
 
 typedef enum {
+    // TODO: Add more meaningful return types
     IRIS_HAL_OK = 0,
-    IRIS_HAL_ERROR = 1,
-} IrisHALReturn;
+    IRIS_HAL_FAILURE = 1,
```

Comment:
For example:
- Failure: OBC sends command, but it is NACK for some reason
- Error: OBC sends command, but it is neither NACKed or ACKed. Also if there is any issues on the obc side like writing into SD card, we should return IRIS_HAL_ERROR

Reply:
Noted. I am going to open be opening a new PR addressing Iris service later this week. I will make changes to error catchment in that as this will be closely associated with the iris service